### PR TITLE
Story/hficalc historic requests/2806

### PR DESCRIPTION
**Pull request by andrea-williams**
To confirm HFI Calc works when loading saved historical HFI requests:
I've created an HFi request for Coastal FC for the current prep period (May 1-5). When you load the page you should see Fraser Zone marked as ready with only 2 stations selected, and Pemberton Zone will have non-default fire starts set.
There's also a stored HFI request for COFC for April 20-23, with Fraser Zone marked as ready and non-default fire start values set. 

Once this PR is deployed to prod & merged, we'll be able to test the Alembic migration where the Honna station gets moved from Fraser Zone to a new planning area called Haida Gwaii, and confirm that historic hfi requests can still be displayed. 

# Test Links:
[Landing Page](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2810.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: api/app/routers/hfi_calc.py [hunk 1]
```diff
@@ -347,9 +345,9 @@ async def get_hfi_result_with_date(fire_centre_id: int,
                 date_range = DateRange(start_date=start_date, end_date=end_date)
             else:
                 date_range = None
-            request, _, fire_centre_fire_start_ranges = get_prepared_request(session,
-                                                                             fire_centre_id,
-                                                                             date_range)
+            request, request_loaded, fire_centre_fire_start_ranges = get_prepared_request(session,
```
Comment (conbrad) [inline api/app/routers/hfi_calc.py:348]:
```suggestion
            request, _, fire_centre_fire_start_ranges = get_prepared_request(session,
```

File: api/app/tests/hfi/test_hfi_endpoint_request.feature [hunk 2]
```diff
@@ -13,7 +13,6 @@ Feature: /hfi/
             | url                                               | status_code | response_json                            | request_saved | stored_request_json                   |
             | /api/hfi-calc/fire_centre/1                       | 200         | hfi/test_hfi_endpoint_load_response.json | False         | None                                  |
             | /api/hfi-calc/fire_centre/1                       | 200         | hfi/test_hfi_endpoint_load_response.json | False         | test_hfi_endpoint_stored_request.json |
-            | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25 | 200         | hfi/test_hfi_endpoint_load_response.json | False         | None                                  |
```
Comment (conbrad) [inline api/app/tests/hfi/test_hfi_endpoint_request.feature:16]: Why do we need to delete just this test?

Comment (conbrad) [reply api/app/tests/hfi/test_hfi_endpoint_request.feature:16]: I think we can add this test back, it passes locally for me

Comment (andrea-williams) [reply api/app/tests/hfi/test_hfi_endpoint_request.feature]: Added back in

**Review by conbrad — APPROVED** [review]

**Review by dgboss — APPROVED** [review]
LGTM

**Pull request merged**